pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -8,14 +8,16 @@ CREATE SCHEMA IF NOT EXISTS SCHEMA_METRIC;
 
 
 CREATE EXTENSION IF NOT EXISTS timescaledb WITH SCHEMA public;
+
+CREATE DOMAIN SCHEMA_PROM.label_ids AS int[] NOT NULL;
 -----------------------
 -- Table definitions --
 -----------------------
 
 CREATE TABLE SCHEMA_CATALOG.series (
     id bigserial PRIMARY KEY,
     metric_id int,
-    labels int[],
+    labels SCHEMA_PROM.label_ids,
     UNIQUE(labels) INCLUDE (id)
 );
 CREATE INDEX series_labels_id ON SCHEMA_CATALOG.series USING GIN (labels);
```

Comment:
Does this need to be on `labels::int[]`?

Reply:
I don't think so, in any case it seems to work and doesn't complain.